resources/lib/library.py

```diff
@@ -349,7 +349,7 @@ def startup(self):
                 if settings('kodiCompanion.bool'):
 
                     for plugin in self.server['api'].get_plugins():
-                        if plugin['Name'] in ("Jellyfin.Kodi Sync Queue", "Kodi companion"):
+                        if plugin['Name'] in ("Kodi Sync Queue", "Kodi companion"):
```

Comment:
This is an unrelated change. It should be made in a separate PR. And I'm also thinking we should remove this check altogether as it's pretty silly. If fast sync is enabled, it should just try to do it and show an appropriate error message if it fails.

Reply:
Weird, this commit shouldn't be here, I changed this to get rid of the error everytime I try to start Kodi :-)